src/Payum/Stripe/Resources/views/Action/obtain_checkout_token.html.twig

```diff
@@ -10,7 +10,8 @@
             data-image="{{ model.image|default("") }}"
             data-name="{{ model.name|default("") }}"
             data-description="{{ model.description|default("") }}"
-            data-amount="{{ model.amount }}">
+            data-amount="{{ model.amount }}"
+            data-currency="{{ model.currency }}">
```

Comment:
Good call. I wasn't sure whether the currency would always be set on the model or not but adding the default gets round that (and doesn't break existing behaviour). Have amended pull request.